### DISCUSSION TITLE
No Animation Triggered for Translate RenderTransform

### DISCUSSION THREAD

**Discussion by NeotericDev** [AvaloniaUI/Avalonia#16958] [Q&A] [answered]
Hello Everyone,

Am learning AvaloniaUI and its animation system. And noticed that am not able to Animate TranslateTransform.Y. Instead the Border moves the position instantly.

I hope am not doing it wrong.

**Code Snippet 1 :** 
![image](https://github.com/user-attachments/assets/feb60080-797f-426d-bdcd-437eaec84d14)

### The Above Code does change the position of the Border. But I observe below issues 

1. No smooth Animation is happening.
2. Upon clicking the CheckBox **Odd-Number of times** the Border gets repositioned without Animation but immediately _without any delay as opposed to the Duration of 0.5 seconds_ provided.
3. Upon clicking the CheckBox **Even-Number of times** the Border gets repositioned without Animation but **_with a delay 0.5 seconds as provided_**.

I guess am doing something wrong here. If not Can I report it as a Bug?

**Comment by timunie** [comment, marked as answer]
I think you can only have one RenderTransform at a time, so use a converter or a TransformGroup (may be named a bit different)

**Reply by NeotericDev** [reply]
Thanks much @timunie for the tip of using a converter, It worked like a Charm. 

But the XAML version should be working with TranslateTransform.y Setter property, not sure why it isn't working. Probably a Bug which needs attention?
